### ISSUE TITLE
Network Graph, Render with Live Mm Data

### ISSUE THREAD

**Issue by adonahue** [rifflearning/zenhub-platform-project#72]
This story is to allocate time to updating the network visualization to use live Mm data, and update in real time. 

- [x] Find out how to access the Mm data, via api, and reusable for other graphics.
- [ ] Hook up the data to the visualization.

**Comment by brecriffs** [comment]
@adonahue This might be complete?

**Comment by adonahue** [comment]
@brecriffs - I think as a proof of concept it is. The code all still needs to get merged though, yeah?

**Comment by brecriffs** [comment]
You're right. After reconsidering, I think the first one is definitely done, but the second has not been merged yet. It's possible we should start a new card or set of cards to track the new and fully(almost) vetted task.

**Comment by adonahue** [comment]
Agree @brecriffs - I will write new stories for the updated UI on the graphic and the rules for all the queries. But I think once all relevant code for this story is merged, we can consider it done.

**Comment by adonahue** [comment]
This story is accepted as is, based on the proof of concept code that was deployed on DEV. A new series of stories for this epic have been written and estimated to complete production-level work.
